### DISCUSSION TITLE
Why are termux.net/bootstrap/ and termux.net inaccessible?

### DISCUSSION THREAD

**Discussion by Contemporaries** [termux/termux-app#2854] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/51323917/174958590-1eef14c7-f492-4a9a-9e9f-4916b9a3e406.png)

**Comment by Contemporaries** [comment]
The make my old android(6.0) unable to install for termux0.72

**Comment by ghost** [comment, marked as answer]
Support for Android 5/6 is re-enabled as in:
https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6

**Reply by Contemporaries** [reply]
Thank you!
